docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,14 @@
 Version history
 ===============
 
+5.3.1 (2016 December 16)
+-----------------------
+
+Java bug fixes:
```

Comment:
I'd be inclined to replace it with file format fixes (or other as appropriate down the line)